packages/service-provider-core/src/index.ts

```diff
@@ -30,5 +31,7 @@ export {
   CliOptions,
   generateUri,
   Scheme,
-  DEFAULT_DB
+  DEFAULT_DB,
+  ServiceProviderCore,
+  bson
```

Comment:
We export this here because there are places we need to use the BSON package directly and don't care if it's modified, for example `calculateObjectSize` is a method that exists on the `bson` package but not on the driver, or where the `cli-repl` needs to instantiate an `ObjectId` for logging purposes, which happens before the ServiceProvider is instantiated. It's also useful for tests, so we can completely get rid of requiring the `bson` package from anywhere other than `service-provider-core`.